recipe/conda_build_config.yaml

```
@@ -12,8 +12,7 @@ ctng_cpu_arch:
 #ctng_gdb:     # [not ppc64le]
 #  - 7.12.1    # [not ppc64le]
 ctng_gcc:
-  - 7.5.0  # [linux64 or linux32 or aarch64]
-  - 8.4.0  # [ppc64le]
+  - 9.3.0
```

We can keep building multiple versions in the same branch.

```suggestion
  - 7.5.0  # [linux64 or linux32 or aarch64]
  - 8.4.0  # [ppc64le]
  - 9.3.0
```

---

🤦